### PR TITLE
Add marginGetOrder

### PR DESCRIPTION
**Pull request by krolov**
Hi @balthazar 
I hope you are doing good.

I add a new method - marginGetOrder.

Also, I fixed several things:
- Add marginOrder description to README.md
- Remove double assetDetail from http-client.js
- For isIsolated field, I add `string | boolean` type. According to Binance documentation it must be string, but Binance kindly accepts boolean too. So I keep boolean for back compability.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,6 +107,8 @@ Following examples will use the `await` form, which requires some configuration
   - [marginCreateIsolated](#marginCreateIsolated)
   - [marginIsolatedTransfer](#marginIsolatedTransfer)
   - [marginIsolatedTransferHistory](#marginIsolatedTransferHistory)
+  - [marginOrder](#marginorder)
+  - [marginGetOrder](#margingetorder)
```
Comment (balthazar) [inline README.md:111]: could you make the link camelcase too?

File: README.md [hunk 2]
```diff
@@ -2483,6 +2485,137 @@ console.log(await client.marginIsolatedTransferHistory({ symbol: 'BNBUSDT'}));
...
+| ----------------- | ------- | -------- | ------------------------- |
+| symbol            | String  | true     | asset, such as `BTC`      |
+| isIsolated        | String  | false    | for isolated margin or not, `TRUE`, `FALSE`, default `FALSE`
+| side              | String  | true     | `BUY` `SELL` |
+| type              | String  | true     |  
+| quantity          | String  | false    |
+| quoteOrderQty     | String  | false    |
+| price             | String  | false    | 
+| stopPrice         | String  | false    | Used with `STOP_LOSS`, `STOP_LOSS_LIMIT`, `TAKE_PROFIT`, and `TAKE_PROFIT_LIMIT` orders.
+| newClientOrderId  | NuStringmber  | false    | A unique id among open orders. Automatically generated if not sent.
```
Comment (balthazar) [inline README.md:2509]: invalid type

File: src/http-client.js [hunk 3]
```diff
@@ -361,7 +361,6 @@ export default opts => {
     accountSnapshot: payload => privCall('/sapi/v1/accountSnapshot', payload),
     universalTransfer: payload => privCall('/sapi/v1/asset/transfer', payload, 'POST'),
     universalTransferHistory: payload => privCall('/sapi/v1/asset/transfer', payload),
-    assetDetail: payload => privCall('/sapi/v1/asset/assetDetail', payload),
```
Comment (balthazar) [inline src/http-client.js:364]: good catch!

**Review by balthazar — CHANGES_REQUESTED** [review]
Just minor tweak on the readme, but everything looks good otherwise! Thanks!

**Comment by krolov** [comment]
@balthazar  fixed

**Comment by krolov** [comment]
Also I made `transactTime` for `Order` optional. It exists only if this object returns after new order functions.

**Comment by balthazar** [comment]
Sounds good, thanks again :)

**Pull request merged**